README.md

```diff
@@ -110,26 +116,27 @@
 
 ## Tutorials
 
-### MDN Docs
+## Hugging Face Transformers
 
-- [WebNN](https://developer.mozilla.org/en-US/search?q=webnn) - To do
+- To do
 
-### WebNN API
+### Google JS ML Framework
```

Comment:
Do you mean TensorFlow Lite? If we don't have clear target, we can remove it in this PR and add it later.